src/homekit.cs

```diff
@@ -1416,4 +1528,211 @@ interface HMCameraAudioControl
 		[NullAllowed, Export ("volume", ArgumentSemantic.Strong)]
 		HMCharacteristic Volume { get; }
 	}
+
+	[Watch (4,0), TV (11,0), iOS (11,0)]
+	[BaseType (typeof(HMTimeEvent))]
+	[DisableDefaultCtor]
+	interface HMCalendarEvent : NSCopying, NSMutableCopying {
+
+		[Export ("initWithFireDateComponents:")]
+		IntPtr Constructor (NSDateComponents fireDateComponents);
+
+		[Export ("fireDateComponents", ArgumentSemantic.Strong)]
+		NSDateComponents FireDateComponents { get; }
+	}
+
+	[Watch (4,0), TV (11,0), iOS (11,0)]
+	[BaseType (typeof(HMCalendarEvent))]
+	[DisableDefaultCtor]
+	interface HMMutableCalendarEvent {
+
+		[Export ("initWithFireDateComponents:")]
+		IntPtr Constructor (NSDateComponents fireDateComponents);
+
+		[Export ("fireDateComponents", ArgumentSemantic.Strong)]
+		NSDateComponents FireDateComponents { get; set; }
```

Comment:
Sounds like it could be indeed, I did not know of this attribute, never had to use it. Sounded weird to me that the generated code did not have override.